src/drivers/driver_zephyr.c

```diff
@@ -366,6 +366,12 @@ void wpa_drv_zep_event_proc_scan_done(struct zep_drv_if_ctx *if_ctx,
 			event);
 }
 
+void wpa_drv_zep_event_proc_scan_abort(struct zep_drv_if_ctx *if_ctx)
```

Comment:
This can simply be SCAN_DONE, WPA supplicant doesn't treat this any differently.

Reply:
Hi @krish2718  this is the callback function from driver side and it's different case from scan_done.

Reply:
ok, but how does this plugin to the WPA supplicant? apart from acting as an intimation of SCAN_DONE?

Reply:
Hi @krish2718 I have updated the wpa_drv_zep_event_proc_scan_abort() function.
This is triggered by driver, as the driver's scan is in progress and aborted (maybe by "wifi disconnect" cmd), driver should notify supplicant, and supplicant can release the scan_work, don't let the next connection is blocked by the scan_work. 
scan done is all the channel scanned over, but scan abort is scan aborted in progress

Reply:
Sure, I understand, as you say it's main purpose is to make the scan free in WPA supplicant which can be done in two ways:

1. Send SCAN_DONE even for driver initiated abort
2. WPA supplicant itself aborts the scan, e.g., disconnect

For #1 I don't think we need to distinguish if it's regular SCAN_DONE or SCAN_DONE from driver abort.

Reply:
okay, I will check and remove this part